ecosystem/sep-0030.md

```diff
@@ -526,7 +549,78 @@ See [Common Fields].
     { "role": "sender", "authenticated": true },
     { "role": "receiver" },
   ],
-  "signer": "GADF..."
+  "signer": "GADF...",
+  "signers": [
+    { "signer": "GADF...", "generated_at": "2006-01-02T15:04:05Z" }
+  ]
+}
+```
+
+#### `POST /accounts/<address>/sign/<signing-address>`
+
```

Comment:
In the case where I'm replacing `signerOLD` with `signerNEW` should I perform two requests to `POST /accounts/<address>/sign/<signing-address>`? 

In that case, I'm assuming it's safer to add `signerNEW` prior to removing `signerOLD` so we prevent a scenario where the user is left without signers.

Reply:
This endpoint is for signing transactions and the client should choose a signing address that they already have registered as a signer of the account. When the client gets around to updating a signer they can add the new and remove the old signers in a single transaction on the network, and then use the new signing key with the recovery server. I think they are the only constraints. The server won't ever remove the old signing key, unless we add that functionality in the future.